src/components/ui/Label.tsx

```diff
@@ -5,6 +5,8 @@
  * shadcn/ui: https://ui.shadcn.com/docs/components/label
  * Radix UI: https://www.radix-ui.com/primitives/docs/components/label
  */
+
+'use client';
```

Comment:
Since it's using radix UI, Label component causes error without "use client"

Reply:
I updated InputUsageExample.ts, by adding "Label" component in there.
Could you try render InputUsageExample in your local env and check if you get this error or not, please?
In my env it does cause this error, and "use client" in Label component will resolve the error 
<img width="339" alt="Screenshot 2023-12-16 at 19 48 10" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/99339182/df4f7493-5f3f-43c0-be12-203299cb1d50">

Reply:
Thank you for checking it. It happened in my environment, too. It appears that the primitive label component utilizes the `onMouseDown` event internally.

Here is the source:
https://github.com/radix-ui/primitives/blob/main/packages/react/label/src/Label.tsx

Reply:
Thank you for sharing the source! I should have put it in the first place... sorry! 
So I'll leave the "use client" as it is.